### DISCUSSION TITLE
Small Taskbar Icons

### DISCUSSION THREAD

**Discussion by tduffy83** [valinet/ExplorerPatcher#90] [Ideas]
Can you add the option to toggle small taskbar icons?  It's a simple registry key.  [HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Explorer\Advanced]
"TaskbarSmallIcons"  Currently the patcher restores most of the functionality I liked from Windows 10, but is missing this key feature.

**Comment by valinet** [comment]
Implemented in 22000.258.26.3.

**Reply by tduffy83** [reply]
There isn't an item in the GUI to configure, at least I couldn't find.

**Reply by valinet** [reply]
![image](https://user-images.githubusercontent.com/6503598/137369688-b4f3fb23-864f-4c26-850f-b3633bc61488.png)

**Reply by valinet** [reply]
I think it is there. Please, I don't want to be the rude one, but let's pay a bit more attention to stuff in general.

**Reply by tduffy83** [reply]
It is in fact there.  Despite downloading what github told me was the latest version just yesterday, somehow I got the 258.26.2 version instead of the latest 258.26.3 version.  I have upgraded to the actual latest version and all is well.  

And as an aside, usually when you have to preface something with "I don't want to be the rude one" whatever comes next will in fact be rude.  I understand you do this for free and don't owe anything to anyone, but oftentimes mistakes happen, even to people who are paying attention.  At any rate, no harm taken and I really appreciate this work, it allows me to easily use Windows 11 and the benefits it has while keeping elements I relied upon from Windows 10.

**Reply by valinet** [reply]
Yeah, you are right, I was in a bad mood, you are totally right. I apologize for that. I made myself a promise, I hope I will keep it, that behaviors like these from me are a thing of the past from now on. Again, I deeply apologize, I guess each time is a good time to learn something new, change and improve.

And indeed, maybe the browser played around and served you a cached version. Sorry again and I hope that I did not make you too mad on me. Thanks.
